### PR TITLE
Copying in the latest ChefDK 3 omnibus definitions to fix the Workstation build

### PR DESCRIPTION
**Pull request by tyler-ball**
### Description

Getting ready for Workstation 0.3 release

### Issues Resolved

N/A

### Check List

- [x] PR title is a worthy inclusion in the CHANGELOG
- [ ] You have locally validated the change
- [x] `www/site/content/docs/` has been updated with any relevant changes:
  - * new or changed error messages in 'troubleshooting.md'
  - * new or changed CLI flags in cli-reference.md

### CODE REVIEW

File: omnibus/config/projects/chef-workstation.rb [hunk 1]
```diff
@@ -67,17 +67,18 @@
 dependency "rust-uninstall"
 
 dependency "chef-dk"
+dependency "chef-dk-gem-versions"
 
 dependency "gem-permissions"
 dependency "rubygems-customization"
+dependency "shebang-cleanup"
```
Comment (tyler-ball) [inline omnibus/config/projects/chef-workstation.rb]: I'm so sad it took me a week to figure this out. Any idea why we did not include this dependency originally @jonsmorrow @marcparadise @robbkidd ?

File: omnibus/config/projects/chef-workstation.rb [hunk 2]
```diff
@@ -67,17 +67,18 @@
...
 
 if windows?
   dependency "chef-dk-env-customization"
   dependency "chef-dk-powershell-scripts"
 end
 
 dependency "version-manifest"
-dependency "clean-static-libs"
```
Comment (tas50) [inline omnibus/config/projects/chef-workstation.rb]: ?

Comment (tyler-ball) [reply omnibus/config/projects/chef-workstation.rb]: Copied from https://github.com/chef/chef-dk/blob/v3.11.1/omnibus/config/projects/chefdk.rb#L69

Comment (tas50) [reply omnibus/config/projects/chef-workstation.rb]: Seems like we want that but if we're not doing it in DK we should nuke it for now

Comment (tyler-ball) [reply omnibus/config/projects/chef-workstation.rb]: I searched for a while through a few different means and could not figure out when that line got removed. I think it got dropped during crazytown days.

Comment (tas50) [reply omnibus/config/projects/chef-workstation.rb]: https://github.com/chef/omnibus-software/pull/1058 this cleans it up by adding the old logic into our new ruby cleanup def

**Pull request merged**